### ISSUE TITLE
Lists

### ISSUE THREAD

**Issue by gerwitz** [microdotblog/issues#73]
I use Twitter lists to "tag" other users and would like to do this on micro.blog. It's important that they can be either public or private.

Being one-way, they avoid a lot of the moderation difficulties of "groups" but I believe a user should also have the right to "block" a list and prevent their username from appearing on it. I'm sure there are other vulnerable user concerns I haven't thought of.

**Comment by ghost** [comment]
To be clear: the feature request is for creating account-based lists?

**Comment by gerwitz** [comment]
Indeed. I now worry that if these were public they could become a proxy for follows and lead to popularity incentives, but I’d still like private lists.

I didn’t mention above, but a key feature of these lists on Twitter is they can be used to view timelines. It serves as a way to get the reading benefit of multiple accounts, without creating multiple identities.

**Comment by ghost** [comment]
Agreed. Whilst this is sort-of possible with feed readers, the fact is that Micro.blog feeds have already become more than just basic blog feeds and so this kind of social-based feature ought to be seriously considered.

I would use this feature and it would certainly make it easier for either specific or heavy use of the social network side of Micro.blog.

**Comment by issimonwoods** [comment]
For clarity: the ghost user is my old account.

This feature would be useful for events like [Micro Camp](https://camp.micro.blog/) and could generally help with the much-discussed issue of “discovery”.
